cnxepub/tests/test_formatters.py

```diff
@@ -10,25 +10,124 @@
 import os
 import sys
 import unittest
+from lxml import etree
```

Comment:
Yeah, I accidentally moved that. (I duped it, then flake8 caught the double import, and I removed the wrong one) I'll fix.